### ISSUE TITLE
Unexpected error during neural training

### ISSUE THREAD

**Issue by gdaine** [ModernMT/MMT#341]
I've been using MMT for several months now. I just upgraded (clean install) to 2.3 and started to build a new neural engine from scratch with the new build, and got the following error message:

=========== TRAINING STARTED ===========

ENGINE:  trneural
BILINGUAL CORPORA: 4 documents
MONOLINGUAL CORPORA: 0 documents
LANGS:   en > es

INFO: (1 of 8) Corpora cleaning...                                     DONE (in 1m 15s)
INFO: (2 of 8) Corpora pre-processing...                               DONE (in 15s)
INFO: (3 of 8) Context Analyzer training...                            DONE (in 4s)
INFO: (4 of 8) Aligner training...                                     DONE (in 1m 38s)
INFO: (5 of 8) Creating translation memory...                          DONE (in 30s)
INFO: (6 of 8) Preparing training data...                              DONE (in 1m 56s)
INFO: (7 of 8) Neural decoder training...                             ERROR Unexpected exception:
	TypeError("__init__() got an unexpected keyword argument 'dim'",)
=========================
Do you have any idea what could have caused this? I'm using Ubuntu 16.04 with a GeForce GTX 1060.

Thanks,
Gary

**Comment by davidecaroselli** [comment]
Hi @gdaine 

new versions of MMT require PyTorch 0.3. Please follow the instructions provided in the file [INSTALL.md](https://github.com/ModernMT/MMT/blob/master/INSTALL.md#pre-installation-actions). I suggest also to update your CUDA driver to the latest release.

Best,
Davide

**Comment by davidecaroselli** [comment]
I close this issue for inactivity, if you need support please don't hesitate to re-open it!

Best,
Davide
